src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -336,22 +338,20 @@ function getProjectDependencies(buildCommand: CodeTransformMavenBuildCommand, mo
         '-Dmdep.useRepositoryLayout=true',
         '-Dmdep.copyPom=true',
         '-Dmdep.addParentPoms=true',
+        '-X',
+        '-e',
```

Comment:
Added these two flags since they will include more error logging in the command output